### ISSUE TITLE
No new page request but still have PageSize and ScrollOffset?!

### ISSUE THREAD

**Issue by dzpt** [froala/wysiwyg-editor#3667]
I see these parameters here:
https://www.froala.com/wysiwyg-editor/docs/concepts/image/manager

But after checking, there is no new page request and no param is determined to request new page?

and i found this: https://github.com/froala/wysiwyg-editor/issues/1564
so i suppose there is no physical page load till now (2019) , i have to return full list of images which cause performance issue !?!

**Comment by casualuser** [comment]
Hello,

Image manager is sending new page request as you're scrolling images thumbs list in the Image manager.
This parameter allows you to tweak conditions for fetching new page 
- https://www.froala.com/wysiwyg-editor/docs/options#imageManagerScrollOffset

But this is generally how it works with simple Froala temporary image storage. If you really want to manage your images intelligently then you can build different custom flows with Image methods and events combinations.

Also you can customize server side for handling Images with different SDKs like node.js, python, etc
https://www.froala.com/wysiwyg-editor/docs/sdks/nodejs/image-manager
https://www.froala.com/wysiwyg-editor/docs/sdks/python/image-manager
https://www.froala.com/wysiwyg-editor/docs/sdks

**Comment by aaveidt** [comment]
I know how it works, after deploying my own backend , there is no new page request to the api after i reach to the end of the modal popup.

If you want to get it work, after reached the ScrollOffset, your editor must send the new request to server to get new data like : `?page=2`. But nothing happended

**Comment by tisdall** [comment]
#1564 implies that the `imageManagerLoadURL` is supposed to return the entire list of images and the UI is supposed to just fetch the first `imageManagerPageSize` of images and load more after scrolling down.  However, for me, the window seems to load all images immediately regardless of that setting.

**Comment by aaveidt** [comment]
@tisdall yes, i knew it. it's a redundant plugin since it loads whole img json file without pagination.

**Comment by ogaponcic** [comment]
@tisdall same problem for me =(
any updates here? do you find a proper solution? workaround ?
![froalamanager](https://user-images.githubusercontent.com/15216460/76338574-8b375d80-6301-11ea-9ffa-dd5eb300adf8.png)
almost 900 images are loaded each time i open the gallery plugin.

**Comment by shafikhaan** [comment]
@ogaponcic did you found any solution for it ?

**Comment by jungar1111** [comment]
On Mon, May 4, 2020 at 8:25 PM, Shafee Khan<notifications@github.com> wrote:   


@ogaponcic did you found any solution for it ?

**Comment by ilyaskarim** [comment]
We’re closing this issue due to inactivity. If you’re still experiencing this problem on the latest version of our editor, please feel free to reopen the issue and let us know. You can find details on the latest updates here: [Froala Editor Changelog](https://froala.com/wysiwyg-editor/changelog/).
